src/State.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
 
+#include <Tracy/Tracy.hpp>
```

Comment:
Can remove `#include <Tracy/Tracy.hpp>`, it's already included in `TracyD3D11.hpp`

Reply:
Just kidding :)